xtask/src/lib.rs

```diff
@@ -433,6 +433,35 @@ pub fn bump_version(workspace: &Path, package: Package, amount: Version) -> Resu
     manifest["package"]["version"] = toml_edit::value(version.to_string());
     fs::write(manifest_path, manifest.to_string())?;
 
+    for pkg in Package::iter() {
+        if [package, Package::HilTest, Package::Examples].contains(&pkg) {
```

Comment:
Nit pick: Shall we filter these with `::iter().filter()`, instead of this inside the loop?